webapp/src/components/LinkedChannelCard/LinkedChannelCard.component.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import {Icon} from 'components/Icon';
+
+import {LinkedChannelCardProps} from './LinkedChannelCard.types';
+
+import './LinkedChannelCard.styles.scss';
+
+export const LinkedChannelCard = ({msTeamsChannelName, msTeamsTeamName, mattermostChannelName, mattermostTeamName}: LinkedChannelCardProps) => (
```

Comment:
This can be fixed after we update the lint rules present in the webapp setup automatically. This was also help us in fixing the import order to avoid fixing it manually)